### DISCUSSION TITLE
学习时长

### DISCUSSION THREAD

**Discussion by wxm78** [liuyunfz/chaoxing_tool#11] [Q&A]
大佬，学习时长刷不了了。

**Comment by liuyunfz** [comment]
您好，感谢您的反馈。经过测试，目前刷取视频学习时长功能正常。使用此功能您需要了解的是：使用本功能将自动开启多线程对多个课程进行同步刷取时长；每次刷取以一分钟为间隔向服务器发取请求，所以你需要间隔一分钟才能看到学习时长的变化。如果上述都不是您说遇到的问题，则是否是您的单个课程遇到了类似“很抱歉，您没有权限访问这个页面”的错误。如果以上都没有囊括，也烦请您给予我更多的信息。
